models/userDao.js

```diff
@@ -0,0 +1,125 @@
+const { appDataSource } = require('./datasource')
+const middleErr = require('../middleware/error')
+const sqlResult = require('../util/sqlResult')
+const { hash } = require('bcrypt')
+
+const existCheck = async( userEmail ) => {
+    try{
+        const result = appDataSource.query(`
+            SELECT * FROM users WHERE email = ?
+        `, [ userEmail ]
+        )
+        return result
+    }catch(err){
+        console.log(err)
+        middleErr.error(500, 'INVALID_DATA_INPUT')
+    }
+}
+const verifyUser = async( userId, userEmail, admin_status ) => {
+    try{
+        const result = await appDataSource.query(`
+            SELECT * FROM users
+            WHERE id = ? AND email = ? AND admin_status = ?
+        `, [userId, userEmail, admin_status])
+        return result
+    }catch(err){
+        console.log(err)
+        middleErr.error(500, 'INVALID_DATA_INPUT')
+    }
+}
+
+const createUser = async ( userEmail, hashedUserPassword, userNickname, userPhoneNumber, userBirthDay, userGender ) => {
+    try{
+        const create = await appDataSource.query(`
+            INSERT INTO users(email, password, nickname, phone_number, birthday, gender, point)
+            values(?,?,?,?,?,?,?)`,
+            [userEmail, hashedUserPassword, userNickname, userPhoneNumber, userBirthDay, userGender, 1000000]
+        )
+        // 쿼리가 정상 작동했음에도 불구하고 데이터가 수정되지 않은 경우 에러 캐치
+        if(create.insertId=0){
+            middleErr.error(500, 'DATA_INSERT_FAILED')
+        }
+        sqlResult.sqlResult(create)
+        return create
+    }
+    catch(err){
+        console.log(err)
+        middleErr.error(500, 'INVALID_DATA_INPUT')
+    }
+}
+
+const list = async() => {
+    try{
+        return await appDataSource.query(`
+            SELECT * FROM users;
+        `)
+    }catch(err){
+        console.log(err)
+        middleErr.error(500, 'INVALID_DATA_INPUT')
+    }
+}
```

Comment:
```suggestion

const getUserList = async () => {
    try{
        return await appDataSource.query(`
            SELECT * FROM users;
        `)
    }catch(err){
        console.log(err)
        middleErr.error(500, 'INVALID_DATA_INPUT')
    }
}
```
1. dao에서 작성하는 함수의 이름은 어떤 Entity를 C, R, U, D 하는지 명확하게 나타내는 것이 좋습니다. 
2. 그래서 create, get(find), update, delete로 시작하는 함수가 대다수입니다.
3. `*` 로 모든 컬럼을 다 가져오시는 것보다, 내가 필요한 컬럼만 정확하게 지칭해서 가져오시는 것이 좋습니다